api/docker-compose.yml

```
@@ -25,5 +23,23 @@ services:
     ports:
       - "4566:4566"  # LocalStack running on localhost:4566
 
+  app:
+    build:
+      context: ..
+      dockerfile: api/Dockerfile
+    ports:
+      - "3010:3010"
+    env_file:
+      - .env
+    volumes:
+      - ../api:/app/api
+      - ../pm2:/app/pm2
+    depends_on:
+      - redis
+      - db
+      - localstack
+    working_dir: /app/api
+    command: ["/bin/sh", "-c", "sleep 5 && npx ts-node script/storage/seedData.ts && npm run build && pm2-runtime start /app/pm2/pm2-dev.json"]
+
```

```suggestion
    command: ["/bin/sh", "-c", "sleep 5 && npx ts-node script/storage/seedData.ts && npm run build && pm2-runtime start /app/pm2/pm2-dev.json"]
```

Why are we hardcoding `pm2-dev.json`? We have a config per env.
Also, why are we running seeData? If someone, uses Docker to deploy on PROD then this step is incorrect.